yarpl/include/yarpl/flowable/EmitterFlowable.h

```diff
@@ -26,25 +26,20 @@ class EmiterBase : public virtual Refcounted {
  * This is synchronous: the emit calls are triggered within the context
  * of a request(n) call.
  */
-template<typename T>
-class EmiterSubscription : private Subscription, private Subscriber<T> {
+template <typename T>
+class EmiterSubscription : public virtual Subscription,
```

Comment:
It doesn't seem like we need `virtual` keyword in here. https://en.wikipedia.org/wiki/Virtual_inheritance